LunaBot/Commands/User/SnugCommand.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Discord.WebSocket;
+using LunaBot.Database;
+
+namespace LunaBot.Commands
+{
+    [LunaBotCommand("Snug","snug")]
+    class SnugCommand : BaseCommand
+    {
+        public override async Task Process(SocketMessage message, string[] parameters)
+        {
+            using (DiscordContext db = new DiscordContext())
+            {
+                ulong userId = message.Author.Id;
+                ulong userId2;
+
+                if (message.MentionedUsers.Count > 0)
+                {
+                    Random random = new Random();
+                    int rand = random.Next(0, 2);
```

Comment:
instead of using your own random property you should use the same system I am using for these. Its a little more complex but it allows for a system that is easier to read. 

Here is an example of the Kick method that uses random sentences for kicks: [KickUserHelper.cs](https://github.com/javierdlg/LunaBot/blob/master/LunaBot/ServerUtilities/KickUserHelper.cs)

Reply:
here is how you'd use it in this method: https://github.com/javierdlg/LunaBot/blob/fb96d1a53a7ea91e360cda19932bd5b1948c1565/LunaBot/Commands/Admin/PurgeCommand.cs#L38